MediaBrowser.Providers/Plugins/Omdb/OmdbProvider.cs

```diff
@@ -459,13 +471,13 @@ private bool IsConfiguredForEnglish(BaseItem item)
             return string.Equals(lang, "en", StringComparison.OrdinalIgnoreCase);
         }
 
-        internal class SeasonRootObject
+        public class SeasonRootObject
```

Comment:
Why are these public now?

Reply:
Access inside tests

Reply:
Expose internals to the tests (like we do with some other projects)